tests/integration/annotation_import/conftest.py

```diff
@@ -291,6 +291,32 @@ def ontology():
     return {"tools": tools, "classifications": classifications}
 
 
+@pytest.fixture
+def wait_for_label_processing():
+    """
+    Do not use. Only for testing.
+
+    Returns project's labels as a list after waiting for them to finish processing.
+    If `project.labels()` is called before label is fully processed,
+    it may return an empty set
+    """
+
+    def func(project):
+        timeout_seconds = 10
+        while True:
+            labels = list(project.labels())
```

Comment:
the best we can do? no other api, to, say, check for labels by some sort of id? or just a heuristics till we come up with smth better?

Reply:
Right, no way to fetch labels by id for a project